pan3d/dataset_viewer.py

```diff
@@ -445,18 +567,27 @@ def _time_index_changed(self) -> None:
             and dataset[da_name] is not None
             and dataset[da_name][t] is not None
         ):
+            d = dataset[da_name].coords[t].dtype
             time_steps = dataset[da_name][t]
             current_time = time_steps.values[t_index]
-            if not hasattr(current_time, "strftime"):
-                current_time = pandas.to_datetime(current_time)
-            self.state.ui_current_time_string = current_time.strftime("%b %d %Y %H:%M")
+
+            if d.kind in ["O", "M"]:  # is datetime
+                if not hasattr(current_time, "strftime"):
+                    current_time = pandas.to_datetime(current_time)
+                current_time = current_time.strftime("%b %d %Y %H:%M")
+            elif d.kind in [
```

Comment:
formatting

Reply:
fixed in [2be407c](https://github.com/Kitware/pan3d/pull/67/commits/2be407cd60a2818d990443271416b24aee94f34c)